docs/vault/guide.md

```diff
@@ -204,6 +204,16 @@ Adding new collateral will increase the collateralization rate.
 
 Go to the Vault navigation item in the sidebar and click on the `Deposit Collateral` button. Then follow the instructions.
 
+**polkadot.js**
+
+1. Go to polkadot.js.org/apps and ensure you have the correct nextwork selected
+2. Click on Developer -> Extrinsics
+3. Select your existing Vault account, the `vaultRegistry` pallet and the `depositCollateral` extrinsic
+4. Enter the collateral asset, e.g., `ForeignAssets.2` as the collateral and the wrapped asset, e.g., `KBTC` as the wrapped currency, and the amount of additional collateral to provide. Example, deposit 1 LKSM = 10^12 Planck (1000000000000).
```

Comment:
@nud3l You are using the LKSM vault as an example in the general guide for operating vaults on Kintsugi. I think we either need to be specific that we use `ForeignAsset` for LKSM or we also mention that in case of native tokens or relay chain we use Token instead.